modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-9_{context}"]
+= Release notes for {pipelines-title} General Availability 1.9
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.11 and later versions.
+
+[id="breaking-changes-1-9_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* With this update, the `Conditions` custom resource definition (CRD) type has been removed. As an alternative, use the `WhenExpressions` instead.
+
+* With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.
+
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. You can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+[id="deprecated-features-1-9_{context}"]
+== Deprecated and removed features
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, `ClusterTasks` are deprecated and planned to be removed in a future release. As an alternative, you can use `Cluster Resolver`.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the usage of the `triggers` and the `namespaceSelector` fields in a single `EventListener` specification is deprecated and planned to be removed in a future release. You can use these fields in different `EventListener` specifications successfully.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the `tkn pipelinerun describe` command does not display timeouts for the `PipelineRun` resource.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, the the `PipelineResource` custom resource (CR) is deprecated. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
+* In the Red Hat OpenShift Pipelines 1.9.0 release, custom image parameters from cluster tasks are deprecated. You must run an Operator-provided cluster task with it's own custom image and create a copy of the cluster task.
```

Comment:
> You must run an Operator-provided cluster task with it's own custom image and create a copy of the cluster task.

You don't have to. If customers are OK with whatever we ship, they can use it out-of-the-box. In the case that they want to use their own image, they need to create a copy of cluster task and then use their own image in that copy.

Reply:
@ppitonak or @rupalibehera - Please confirm if my changes with respect to this comment are ok?

Reply:
Nope, instead of 

>You must run an Operator-provided cluster task with it's own custom image and create a copy of the cluster task.

We should include something like

> As an alternative, you can copy a cluster task and use your custom image in it.

Reply:
Sure, thanks for providing feedback. I have made the change as you suggested. Please have a look.

Reply:
@ppitonak please take a final look here